apps/communications/contacts/views/details/js/boot.js

```diff
@@ -45,6 +45,55 @@ window.onload = function() {
       document.body.classList.remove('hidden');
     });
 
+
+    function getParams() {
+      var params = {};
+      var raw = window.location.search.split('?')[1];
+      var pairs = raw.split('&');
+      for (var i = 0; i < pairs.length; i++) {
+
+        var data = pairs[i].split('=');
+        params[data[0]] = data[1];
+      }
+      return params;
+    }
+
+    function renderContact(id) {
```

Comment:
This name is confusing, since it gets the information from the backend, but does not do the proper render isnt?